src/AccountTile/AccountTile.js

```diff
@@ -0,0 +1,206 @@
+      onClick={() => {
+        menuItemHandler(item.onClick);
+      }}
+      onKeyPress={event => {
+        menuItemHandler(item.onClick, event);
+      }}
+    >
+      {item.label}
+    </MenuItem>
+  ));
+
+  let authStatusIndicator;
+  switch (isAuthenticated) {
+    case true:
+      authStatusIndicator = (
+        <Tooltip
+          title={authenticatedText}
+          placement="top"
+          targetWrapperStyle={{
+            alignItems: 'center',
+            display: 'flex'
+          }}
+        >
+          <CheckCircleIcon color="green" size={16} alt={authenticatedText} />
+        </Tooltip>
+      );
+      break;
+    default:
+      authStatusIndicator = (
+        <Tooltip
+          title={expiredText}
+          placement="top"
+          targetWrapperStyle={{
+            alignItems: 'center',
+            display: 'flex'
+          }}
+        >
+          <ExclamationMarkTriangleIcon size={16} alt={expiredText} />
+        </Tooltip>
+      );
+  }
+
+  return (
+    <Popover
+      open={isOpen}
+      onRequestClose={close}
+      targetContainerStyles={{
+        width: width,
+        display: 'block'
+      }}
+      placement="bottom-end"
+      targetEl={
+        // ---------- Account Tile ----------
+        <StyledAccountTile
+          onClick={toggleIsOpen}
+          onKeyPress={toggleIsOpen}
+          open={isOpen}
+          tabIndex="0"
+        >
+          <StyledContentWrapper>
+            <StyledAvatarContainer>
+              <Avatar src={userThumbnail.url} style={{ width: 60, height: 60 }}>
+                {!userThumbnail.url && userThumbnail.letters}
+              </Avatar>
+              {orgName && (
+                <Avatar
+                  src={orgThumbnail.url}
+                  style={{ width: 30, height: 30 }}
+                >
+                  {!orgThumbnail.url && orgThumbnail.letters}
+                </Avatar>
+              )}
+            </StyledAvatarContainer>
+            <StyledTextWrapper>
+              {orgName && (
+                <StyledP title={orgName} demi>
+                  {orgName}
+                </StyledP>
+              )}
+              <StyledP title={user.fullName || fullName}>
+                {user.fullName || fullName}
+              </StyledP>
+              <StyledP title={user.username} small>
+                {user.username}
+              </StyledP>
+            </StyledTextWrapper>
+          </StyledContentWrapper>
+          <StyledIconWrapper>
+            {authStatusIndicator}
+            {actions.length !== 0 && <HandleVerticalIcon scale={16} />}
+          </StyledIconWrapper>
+        </StyledAccountTile>
+      }
+    >
+      {/* Account item dropdown menu */}
+      <Menu>{actions.length !== 0 && menuItems}</Menu>
+    </Popover>
+  );
+};
+
+AccountTile.propTypes = {
+  /** Actions to be included in the dropdown. Each action should be structured: { label: 'Action Label', onClick: someMethod } */
+  actions: PropTypes.array,
```

Comment:
You can describe this with PropTypes:

```suggestion
  /** Actions to be included in the dropdown. Each action should describe a label and an onClick method. */
  actions: PropTypes.arrayOf(
    PropTypes.shape({
      label: PropTypes.string,
      onClick: PropTypes.func
    })
  ),
```